### PR TITLE
Revert corecode appcast interpolation

### PR DESCRIPTION
**Pull request by vitorgalvao**
This makes updates harder for some systems (https://github.com/Homebrew/homebrew-cask/pull/59330#issuecomment-466545542).

### CODE REVIEW

**Review by reitermarkus — APPROVED** [review]
I think it makes more sense to have the unversioned URL in the appcast, and then actually use the URL it redirects to for the `url`.

**Comment by vitorgalvao** [comment]
@reitermarkus That does make more sense. Changed, though I’m not finding the correct versioned URL for `ocenaudio`.

Pinging @core-code since this will end up being merged after all.

File: Casks/marvin.rb [hunk 1]
```diff
@@ -3,8 +3,8 @@
   sha256 'fce9293100fceacabc700e74cac4c6627074f796acf9b716962f16c0ac4b7ff6'
 
   # amazingmarvin.s3-website-us-east-1.amazonaws.com was verified as official when first introduced to the cask
```
Comment (reitermarkus) [inline Casks/marvin.rb]: Comment needs to be updated.

**Comment by core-code** [comment]
> @reitermarkus That does make more sense. Changed, though I’m not finding the correct versioned URL for `ocenaudio`.

unable to find it either.

> Pinging @core-code since this will end up being merged after all.


thanks

**Pull request merged**